config/packages/messenger.yaml

```diff
@@ -1,14 +1,20 @@
 framework:
     messenger:
-        # Uncomment this (and the failed transport below) to send failed messages to this transport for later handling.
+        # Send failed messages to this transport for later handling.
         failure_transport: failed
 
         transports:
             # https://symfony.com/doc/current/messenger.html#transport-configuration
-            async: "%env(MESSENGER_TRANSPORT_DSN)%"
+            async:
+                dsn: "%env(MESSENGER_TRANSPORT_DSN)%"
+                retry_strategy:
```

Comment:
Does this reflect our response time?
Should we not give it a few extra chances to try and save the job?
Is this a PM decision?
Pros/cons